tests/docker/config/construct-query.sparql

```diff
@@ -657,4 +665,25 @@ CONSTRUCT {
                                                   OPTIONAL { ?preSynaptic     skos:notation   ?preSynapticNotation    . } .
 
     } .
+
+  # Morphology features
+  OPTIONAL {
+    ?id a nsg:NeuronMorphology  ;
+          ^nsg:hasSource / ^nsg:hasTarget ?featureAnnotation    .
+    ?featureAnnotation   a   bmo:NeuronMorphologyFeatureAnnotation  .
+
+    OPTIONAL {
+      GRAPH ?g {
+        ?featureAnnotation    bmo:compartment       ?featureCompartment ;
+                              nsg:hasBody           ?featureBody        .
+        ?featureBody          nsg:isMeasurementOf / rdfs:label ?featureLabel ;
+                              nsg:series            ?featureSeries .
+        ?featureSeries        nsg:statistic         ?featureStat   ;
+                              schema:value          ?featureValue  ;
+                              schema:unitCode       ?featureUnit   .
+
+        FILTER (?featureLabel IN ("Soma Radius", "Total Length", "Total Weight", "Total Depth", "Section Strahler Orders", "Partition Asymmetry")) .
```

Comment:
I'm not 100% sure on this one, but I think if the filter could be done as soon as `?featureLabel` is available to avoid looking trying to look up series and annotation? I don't remember if that's how it plans the query

Reply:
move it higher in the query